subcommands/common.go

```diff
@@ -247,3 +247,10 @@ func FindWritableDirInPath(helperPath string) string {
 	}
 	return ""
 }
+
+func IsWritablePath(path string) string {
```

Comment:
An `Is*` function returning a string rather than a boolean is a little awkward. Looks like you are trying to save a couple of lines of code below, but I'd just make `isWritable` a public function and use that.

Reply:
I concur on this. Other than this your PR LGTM.